foundation-models/openai/src/test/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClientGeneratedTest.java

```diff
@@ -583,4 +583,45 @@ void chatCompletionTool() {
                   }
                   """)));
   }
+
+  @Test
+  void chatCompletionResponseGetMessage() {
+    stubForChatCompletion();
+
+    final var response = client.chatCompletion(new OpenAiChatCompletionRequest("Some text"));
+    final var simpleMessage = response.getMessage();
+
+    assertThat(simpleMessage).isNotNull();
+    assertThat(simpleMessage.toolCalls()).isEmpty();
+    assertThat(simpleMessage.content().items()).hasSize(1);
+    assertThat(simpleMessage.content().items().get(0)).isInstanceOf(OpenAiTextItem.class);
+    assertThat(((OpenAiTextItem) simpleMessage.content().items().get(0)).text())
+        .isEqualTo(
+            "I'm an AI and cannot answer that question as beauty is subjective and varies from person to person.");
+
+    stubForChatCompletionTool();
+
+    final var responseWithToolCall =
+        client.chatCompletion(new OpenAiChatCompletionRequest("Some tool request"));
+    var messageWithToolCall = responseWithToolCall.getMessage();
+
+    assertThat(messageWithToolCall).isNotNull();
+    assertThat(messageWithToolCall.content().items()).hasSize(0);
+    OpenAiFunctionCall functionCallItem =
+        (OpenAiFunctionCall) messageWithToolCall.toolCalls().get(0);
+    assertThat(functionCallItem.getId()).isEqualTo("call_CUYGJf2j7FRWJMHT3PN3aGxK");
+    assertThat(functionCallItem.getName()).isEqualTo("fibonacci");
+    assertThat(functionCallItem.getArguments()).isEqualTo("{\"N\":12}");
```

Comment:
why not make this two tests:
- chatCompletionGetMessage
- chatCompletionToolCallGetMessage